classes/controller/AdminController.php

```diff
@@ -595,34 +595,34 @@ public function initBreadcrumbs($tab_id = null, $tabs = null)
         /* content, edit, list, add, details, options, view */
         switch ($this->display) {
             case 'add':
-                $breadcrumbs2['action']['name'] = $this->l('Add', null, null, false);
+                $breadcrumbs2['action']['name'] = $this->trans('Add');
```

Comment:
No need for domain here? (same question for all other calls)

Reply:
It will put them in the 'messages' domain if there is no domain, see https://github.com/PrestaShop/TranslationToolsBundle/pull/60

Reply:
Ok but wouldn't a domain be better? Or we will change it later? (during a migration maybe)